### ISSUE TITLE
DB2 dialect syntax error TimestampIncrementingTableQuerier

### ISSUE THREAD

**Issue by k1th** [confluentinc/kafka-connect-jdbc#552]
There seems to be a double ";" that upsets the server:

```
ERROR Failed to get current time from DB using Db2 and query 'SELECT CURRENT_TIMESTAMP(12) FROM SYSIBM.SYSDUMMY1;' (io.confluent.connect.jdbc.dialect.Db2DatabaseDialect)
com.ibm.db2.jcc.am.SqlSyntaxErrorException: DB2 SQL Error: SQLCODE=-104, SQLSTATE=42601, SQLERRMC=;;ROM SYSIBM.SYSDUMMY1;END-OF-STATEMENT, DRIVER=4.19.66
```

Probably commit 3f5744e0d4c105f2d6e049d71bfbea4b7e19a8dd

**Comment by arslanakhtar61** [comment]
Hi. Thanks Chris for the fix. I am having same problem while using IBM db2 timestamp column. Appreciate if this could be merged with the next release. I tried to manually edit the code and compile the repository but it fails. Error with pom.xml file etc

**Comment by k1th** [comment]
Haven’t heard from the Confluent people. They will need to merge my PR

**Comment by agskrishna** [comment]
Hi Chris,
 Thanks for fixing and raising a PR. 

Confluent Team,
Is there any ETA when this will be merged?

**Comment by rhauch** [comment]
This issue has already been recently resolved via #642 and backported to the `[5.0.x](https://github.com/confluentinc/kafka-connect-jdbc/blob/5.0.x/src/main/java/io/confluent/connect/jdbc/dialect/Db2DatabaseDialect.java#L65)`, `[5.1.x](https://github.com/confluentinc/kafka-connect-jdbc/blob/5.1.x/src/main/java/io/confluent/connect/jdbc/dialect/Db2DatabaseDialect.java#L64)`, `[5.2.x](https://github.com/confluentinc/kafka-connect-jdbc/blob/5.2.x/src/main/java/io/confluent/connect/jdbc/dialect/Db2DatabaseDialect.java#L64)`, and `[master](https://github.com/confluentinc/kafka-connect-jdbc/blob/master/src/main/java/io/confluent/connect/jdbc/dialect/Db2DatabaseDialect.java#L64)` branches, and will be included in the next CP patch release for these branches.

Closing this PR without merging.
